README.md

```diff
@@ -467,6 +467,7 @@ Table of Contents
   * [everystep-automation.com](https://www.everystep-automation.com/) — Records and replays all steps made in a web browser and creates scripts,... free with fewer options
   * [Gremlin](https://www.gremlin.com/gremlin-free-software) — Gremlin's Chaos Engineering tools allow you to safely, securely, and simply inject failure into your systems to find weaknesses before they cause customer-facing issues. Gremlin Free provides access to Shutdown and CPU attacks on up to 5 hosts or containers.
   * [gridlastic.com](https://www.gridlastic.com/) — Selenium Grid testing with free plan up to 4 simultaneous selenium nodes/10 grid starts/4,000 test minutes/month
+  * [Loadest.io](http://loadest.io/) - Online service for determination an endpoint behavior under both normal and at stress conditions. Up to 10,000 request per second is free. Does not require a domain rights validation for single testing.
```

Comment:
There's no pricing information on this site, and its HTTPS is broken.

Reply:
HTTPS is disabled because it is totally free now.
But okay, i will add monetization and will be back soon

Reply:
Could you also add some sort of domain validation or email verification? This tool really seems dangerous for websites.
Also your TOS seems to be fairly copied and incomplete.
![image](https://user-images.githubusercontent.com/66728880/125020580-bc12d880-e096-11eb-8b70-0fdc378a908d.png)

Reply:
> HTTPS is disabled because it is totally free now.
> But okay, i will add monetization and will be back soon

HTTPS is free, via Let's Encrypt. If for some reason you can't use Let's Encrypt, you could use Cloudflare, which is also free.

There is no excuse for not having HTTPS.

Reply:
Okay, now its https

Reply:
There was some other feedback too to onboard, honestly a tool like this with no form of verification that its testing sites I actually own etc is just looking for trouble. 

As it is, I'll not add it.

Reply:
> Could you also add some sort of domain validation or email verification? This tool really seems dangerous for websites.
There is validation mechanism to keep it simple for new users.
Without domain name validation you can only test website:
a) not in website database (was not tested before by anybody)
b) not in alexa golden million
c) returns valid 200 header

And even if you match this conditions, you can test in only once, for second test you must make a validation. TOC requires a lawyer, i use dummy with added some important points. Anyway i fixed misspellings

Reply:
> There was some other feedback too to onboard, honestly a tool like this with no form of verification that its testing sites I actually own etc is just looking for trouble.
> 
> As it is, I'll not add it.

As i say, there is domain validation mechanism, try to test some popular domain, or any domain twice